scipy/stats/morestats.py

```diff
@@ -2692,6 +2693,18 @@ def mood(x, y, axis=0):
         different length along `axis`.
         If `axis` is None, `x` and `y` are flattened and the test is done on
         all values in the flattened arrays.
+    alternative : {'two-sided', 'less', 'greater'}, optional
+        Defines the alternative hypothesis. Default is 'two-sided'.
+        The following options are available:
+
+        * 'two-sided': the scales of the distributions underlying `x` and `y`
```

Comment:
Isn't it `underlying distribution` instead? I am not a native speaker here so not sure here.

Reply:
"underlying" can be used either as an adjective or as the present-participle form of the verb "underlie".

In "underlying distribution of `x`" it is used as an adjective describing the distribution.

In "distribution underlying `x`", I think it is being used as the present-participle of the verb, like the "manager running the operation". There may be a subtlety here but, in any case, it's not an uncommon usage, e.g. [here](https://www.quora.com/What-is-meant-by-underlying-distribution-in-statistics)

>  In latent variables often we assume there is a normal distribution underlying an ordinal variable...

and [here](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0127824)

> then the self-similarity index of such series should yield the type of stable distribution underlying the process

I think I had "the underlying distribution of `x`" originally, but then I would have several "of"s , and that gets confusing.

Reply:
OK thanks for explaining this to me 👍